.github/workflows/release-cli.yml

```diff
@@ -0,0 +1,21 @@
+name: Release CLI Tool
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    name: Build and Publish CLI Tool
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
@Sammers21 let's use `actions/checkout@v2` it's faster and has several bug fixes